app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -86,14 +90,23 @@ class NodeItem(
     var fullName = this.name
 
     while (parent != null) {
-      fullName = "${parent.name} - $name"
+      fullName = "${parent.name} - $fullName"
       parent = parent.getParent()
     }
 
     return fullName
   }
 }
 
+class ComposeNode(
```

Comment:
To make it easier to add a compose screen - with compose we'll have to have separate composables for each story anyways - rather than having to provide a separate story class everywhere we can just define the composable in the registry (similar to how we set the layout id)

Reply:
This is exactly how Compose is designed to be used :)